public/locations-list-map.js

```diff
@@ -708,32 +710,32 @@ function centerMapToNearestMarkers(map, markers, fallbackBounds) {
 
 
 function addMarkerToMap(map, latitude, longitude, address, name, instructions, accepting, open_accepted) {
-    // Text to go into InfoWindow
-    var contentString =
-        '<h5>' + name + '</h5>' +
-        `<div class="label">${$.i18n('ftm-maps-marker-address-label')}</div><div class=value>` + address + '</div>' +
-        `<div class="label">${$.i18n('ftm-maps-marker-instructions-label')}</div><div class=value>` + linkifyHtml(instructions) + '</div>' +
-        `<div class="label">${$.i18n('ftm-maps-marker-accepting-label')}</div><div class=value>` + accepting + '</div>' +
-        `<div class="label">${$.i18n('ftm-maps-marker-open-packages-label')}</div><div class=value>` + open_accepted + '</div>';
-
-    var location = { lat: latitude, lng: longitude };
-    var marker = new google.maps.Marker({
-        position: location,
-        title: name,
-        map: map
-    });
-    // InfoWindow will pop up when user clicks on marker
-    marker.infowindow = new google.maps.InfoWindow({
-      content: contentString
-    });
-    marker.addListener('click', () => {
-      openInfoWindows.forEach(infowindow => infowindow.close());
-      openInfoWindows = [];
-      marker.infowindow.open(map, marker);
-      openInfoWindows.push(marker.infowindow);
-    });
+  // Text to go into InfoWindow
+  var contentString =
+    '<h5>' + name + '</h5>' +
+    `<div class="label">${$.i18n('ftm-maps-marker-address-label')}</div><div class=value>` + address + '</div>' +
+    `<div class="label">${$.i18n('ftm-maps-marker-instructions-label')}</div><div class=value>` + linkifyHtml(instructions) + '</div>' +
+    `<div class="label">${$.i18n('ftm-maps-marker-accepting-label')}</div><div class=value>` + accepting + '</div>' +
+    `<div class="label">${$.i18n('ftm-maps-marker-open-packages-label')}</div><div class=value>` + open_accepted + '</div>';
+
+  var location = { lat: latitude, lng: longitude };
+  var marker = new google.maps.Marker({
+    position: location,
+    title: name,
+    // map: map
```

Comment:
Remove this line too. And can remove the 'map' param. Maybe rename method to 'createMarker', since the add operation has been moved to showMarkers since a day or so ago.